pom.xml

```diff
@@ -95,6 +95,12 @@
 				<groupId>io.zipkin.gcp</groupId>
 				<artifactId>zipkin-sender-stackdriver</artifactId>
 				<version>${zipkin-gcp.version}</version>
+				<exclusions>
+					<exclusion>
+						<groupId>io.zipkin.zipkin2</groupId>
+						<artifactId>zipkin</artifactId>
+					</exclusion>
```

Comment:
Do you also need to add the `<type>` here? Like `<type>test-jar</type>`?

Reply:
Doesn't look like it.

Reply:
Isn't there a core artifact also named 'zipkin' though? https://repo.maven.apache.org/maven2/io/zipkin/zipkin2/zipkin/2.12.4/zipkin-2.12.4.pom

How does it disambiguate between the 'zipkin' core and the zipkin test jar?

Reply:
`<type>` is not allowed under `dependencyManagement`.
In any case, `zipkin-sender-stackdriver` doesn't depend on `zipkin` core.
So, we should be OK. Other deps will bring it in.